libs/utils/android/gfxinfo.py

```diff
@@ -0,0 +1,91 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (C) 2017, ARM Limited and contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Parser for dumpsys gfxinfo output
+
+import ast, re
+
+def get_value(token):
+    try:
+        v = ast.literal_eval(token)
+    except:
+        return token
+    return v
+
+class GfxInfo(object):
+    """
+    Class for parsing and accessing GfxInfo output
+    """
+    __properties = {}
+
+    def __init__(self, filename):
+        """
+        Initialize gfxinfo parser object
+
+        filename: File containing output of gfxinfo
+        """
+        self.filename = filename
+        self.parse_gfxinfo()
+
+    def parse_gfxinfo(self):
+        """
+        Parser for gfxinfo output, creates a name/value pair
+        """
+        with open(self.filename) as f:
+            content = f.readlines()
+
+        for line in content:
+            line = line.rstrip()
+            if not ":" in line:
```

Comment:
Nit: keep quotes consistent (`"` is used here but `'` is used below). Most of the code in LISA prefers `'`.